bindings/nodejs/examples/how_tos/advanced_transactions/advanced_transaction.ts

```diff
@@ -0,0 +1,66 @@
+// Copyright 2023 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+import {
+    AddressUnlockCondition,
+    Client,
+    Ed25519Address,
+    TimelockUnlockCondition,
+    Utils,
+    Wallet,
+    initLogger,
+} from '@iota/sdk';
+require('dotenv').config({ path: '.env' });
+
+// Run with command:
+// yarn run-example ./how_tos/advanced_transactions/advanced_transaction.ts
+
+// This example syncs the account sends outputs with custom features/unlock conditions
```

Comment:
No features shown in the example

Reply:
Yeah was thinking about what the how-to is supposed to show. But you are right. That doesn't matter for the code example

```suggestion
// This example syncs the account and sends an output with a timelock unlock conditions
```